gettext_i18n_rails.gemspec

```diff
@@ -15,7 +15,8 @@ Gem::Specification.new name, GettextI18nRails::VERSION do |s|
   s.add_development_dependency "haml"
   s.add_development_dependency "rake"
   s.add_development_dependency "rails"
-  s.add_development_dependency "ruby_parser", ">= 3"
+  s.add_development_dependency "ruby_parser", ">= 3.7.1"
```

Comment:
can you put a `gem "ruby_parser", ">= 3.7.1"` where it is required in the codebase so nobody runs into weirdness ?